eq-author-api/middleware/import.js

```diff
@@ -1,3 +1,7 @@
+const pinoMiddleware = require("express-pino-logger");
+const pino = pinoMiddleware();
+const logger = pino.logger;
```

Comment:
Not sure if this is the best way to get a hold of the logger, even though it works. This seems to be instantiating express middleware, but the middleware is not being passed into the express handler functions, so it seems a bit wasteful. @tgandrews any ideas if it's possible to extract the logger from expresses middleware somehow? Otherwise feels like the logger is something that we should be exporting in the root `app.js` and passing down (or making available) to the other modules by means of a `getLogger` function for example?

Reply:
Good spot, I didn't notice this change. According to the docs it looks like the middleware exposes `req.log` to get to the logger so we shouldn't have to import it here.